### DISCUSSION TITLE
Feature request: Pagebuilder editor like GrapesJS or Elementor

### DISCUSSION THREAD

**Discussion by vtisnado** [directus/directus#17996] [Feature Requests]
### Summary

Hi guys, first of all, I want to thank the community for all the effort in this project.

I think it will be nice to have a Page Builder like https://grapesjs.com/ this might not be a priority right now, however, I think a group can start working in some integration.

I have to read more the documentation to find the best way to add a feature like this, but if someone can point a direction I can start from there.

Thanks in advance.

### Motivation

This will help to manage page layout/design based in components. User friendly for non tech-savvy web admins and will save a lot of time for devs.

### Detailed Design

Here are some interesting approaches:

1. https://grapesjs.com/demo.html
2. https://www.odoo.com/app/website
3. https://elementor.com/
4. https://microweber.org/

### Must Have:
- Drag and drop
- WYSIWYG
- Preview
- Custom HTML blocks

### Should Have:
- Templates
- Components
- Variables

### Could Have:
- Preview per device (mobile/desktop)

### Won't Have:
- A long learning curve

### Drawbacks

- Development time

### Alternatives

N/A

### Adoption Strategy

This should not affect any existing project

**Comment by connorwinston** [comment]
Hi! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

--- 

Side note, it looks like there has been traction making a Page Builder extension. https://discord.com/channels/725371605378924594/1047807870042910810/1047807870042910810
